app/services/users.js

```diff
@@ -12,3 +15,14 @@ exports.createUser = async payload => {
     throw errors.databaseError('Something went wrong trying to save into the DB');
   }
 };
+
+exports.authenticateUser = email => {
+  try {
+    const token = jwt.sign({ email }, config.common.session.secret, { expiresIn: '6h' });
```

Comment:
Here you could be destructuring this variable at the time of the declaration so as not to have to do so much nesting.